src/backend/database/user.model.js

```diff
@@ -0,0 +1,31 @@
+// user.model.js
+const { DataTypes } = require("sequelize");
+const { sequelize } = require("./connection.js"); 
+
+const User = sequelize.define("User", {
+  id: {
+    type: DataTypes.STRING,
```

Comment:
@juhan26 the type should be `UUID.V4` and don't forget to set default value to auto generated UUIDV4 by sequelize. [see this example](https://sequelize.org/docs/v7/models/data-types/#built-in-default-values-for-uuid) for more detail.